code/obj/machinery/computer/card.dm

```diff
@@ -38,6 +38,26 @@
 	icon = 'icons/obj/computerpanel.dmi'
 	icon_state = "id2"
 
+/obj/machinery/computer/card/portable
+  name = "Portable Id Computer"
```

Comment:
```suggestion
  name = "portable identification computer"
```
Use the same name as the original, fix the casing, prepend "portable".

Reply:
> Use the same name as the original, fix the casing, prepend "portable".

I did know it would become "foldable portable identification computer" but I kind of like how dorky that sounds, idk I've talked with others about it, but if someone else supports that change ill implement it